<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonDetails</class>
 <widget class="QWidget" name="PersonDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#PersonDetails{
	background-color: rgb(33, 33, 33);
}
QPushButton{
	border:2px solid rgb(189, 189, 189); 
	border-radius:12px;
	color: rgb(255, 255, 255);
	height:50px;
	font-size:20px

}
#PersonDetails QPushButton:hover{
	background-color: rgb(28, 113, 216);

}
QPushButton#cancelButton:hover{
	background-color: rgb(255, 0, 0);

}
QLineEdit{
	background-color: rgb(33, 33, 33);
	border:2px solid rgb(189, 189, 189); 
	border-radius:12px;
	color: rgb(255, 255, 255);
	height:50px;
	font-size:20px
}
QLabel#person_picture{
	border:2px solid rgb(189, 189, 189); 
	border-radius:12px;
	padding:5px
}

QLabel { 
	background-color: rgb(33, 33, 33);
	color: rgb(255, 255, 255); 
	height: 50px; 
	font-size: 20px; 
}

QDateEdit { 
	background-color: rgb(33, 33, 33);
	border: 2px solid rgb(189, 189, 189); 
	border-radius: 12px; 
	color: rgb(255, 255, 255); 
	height: 50px; 
	font-size: 20px; 
}


QDateEdit::calendar-popup { 
	border: 2px solid rgb(189, 189, 189); 
	border-radius: 12px; 
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,0">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridPictures"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="resetButton">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="personDetails.qrc">
               <normaloff>:/icons/resources/Icons/rotate-solid.svg</normaloff>:/icons/resources/Icons/rotate-solid.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="selectPicturesButton">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="personDetails.qrc">
               <normaloff>:/icons/resources/Icons/circle-plus-solid.svg</normaloff>:/icons/resources/Icons/circle-plus-solid.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldGrowthPolicy::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>&amp;Name</string>
           </property>
           <property name="buddy">
            <cstring>nameInput</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nameInput">
           <property name="maxLength">
            <number>35</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="birthdayLabel">
           <property name="text">
            <string>&amp;Birthday</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>birthdayInput</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateEdit" name="birthdayInput">
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
           </property>
           <property name="currentSection">
            <enum>QDateTimeEdit::Section::DaySection</enum>
           </property>
           <property name="displayFormat">
            <string>d/M/yyyy</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="phoneNumberLabel">
           <property name="text">
            <string>Phone</string>
           </property>
           <property name="buddy">
            <cstring>phoneNumberInput</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="phoneNumberInput">
           <property name="inputMask">
            <string>+999999999999</string>
           </property>
           <property name="text">
            <string>+000000000000</string>
           </property>
           <property name="maxLength">
            <number>13</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>+000000000000</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="emailLabel">
           <property name="text">
            <string>Ema&amp;il</string>
           </property>
           <property name="buddy">
            <cstring>emailInput</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="emailInput">
           <property name="inputMask">
            <string/>
           </property>
           <property name="maxLength">
            <number>250</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="adresseLabel">
           <property name="text">
            <string>&amp;Adresse</string>
           </property>
           <property name="buddy">
            <cstring>adresseInput</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="adresseInput">
           <property name="maxLength">
            <number>250</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Adresse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="submitButton">
           <property name="text">
            <string>Submit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>birthdayInput</tabstop>
  <tabstop>phoneNumberInput</tabstop>
  <tabstop>emailInput</tabstop>
  <tabstop>adresseInput</tabstop>
 </tabstops>
 <resources>
  <include location="personDetails.qrc"/>
 </resources>
 <connections/>
</ui>
